python/src/py_opcua_variant.h

```diff
@@ -42,6 +42,26 @@ list ToList(const std::vector<SourceType> objects)
   return result;
 }
 
+// This specialisation is needed because on OS-X 10.12 clang-802.0.42
+// with boost 1.59.0 gives the following compile error:
+// /opt/local/include/boost/python/converter/arg_to_python.hpp:209:9:
+//   error: no matching constructor for initialization of
+//      'boost::python::converter::detail::arg_to_python_base'
+//      : arg_to_python_base(&x, registered<T>::converters)
+template <>
+list ToList(const std::vector<bool> objects)
+{
+  list result;
+
+  for (auto obj : objects)
+    {
+      bool tmp = obj;
```

Comment:
To be honest - this is what I think about as template magic. I absolutely don't know why this works as expected. I found another project having the same problem and a patch like the above that solved it (https://github.com/casacore/casacore/blob/master/python/Converters/PycBasicData.h). And no I can't get ride of tmp - it simply does not work anymore if I do.